src/Akka.Streams.Kafka/Stages/CommittableConsumerStage.cs

```diff
@@ -192,23 +137,65 @@ private void MessagesReceived(ConsumerRecord<K, V> message)
             }
         }
 
+        private void HandleOnPartitionsAssigned(IConsumer<K, V> consumer, List<TopicPartition> list)
+        {
+            _partitionsAssigned(list);
+        }
+        
+        
         private void PartitionsAssigned(IEnumerable<TopicPartition> partitions)
         {
             Log.Debug($"Partitions were assigned: {_consumer.Name}");
             _consumer.Assign(partitions);
             _assignedPartitions = partitions;
         }
 
-        private void PartitionsRevoked(IEnumerable<TopicPartition> partitions)
+        private void HandleOnPartitionsRevoked(IConsumer<K, V> consumer, List<TopicPartitionOffset> list)
+        {
+            _partitionsRevoked(list);
+        }
+
+        private void PartitionsRevoked(IEnumerable<TopicPartitionOffset> partitions)
         {
             Log.Debug($"Partitions were revoked: {_consumer.Name}");
             _consumer.Unassign();
             _assignedPartitions = null;
         }
+        
+        private void HandleConsumeError(object sender, Error error)
+        {
+            Log.Error(error.Reason);
+            var exception = new KafkaException(error);
+            switch (_decider(exception))
+            {
+                case Directive.Stop:
+                    // Throw
+                    _completion.TrySetException(exception);
+                    FailStage(exception);
+                    break;
+                case Directive.Resume:
+                    // keep going
+                    break;
+                case Directive.Restart:
+                    // keep going
```

Comment:
Might need to do something different here - for instance, if the issue is that we've lost connectivity to all Kafka hosts an we have to restart the client we'd want to fail this and get rebuilt, I suspect. Maybe the best place to handle that is the default supervision directive?

Reply:
I guess so - did not change this error handling logic, but indeed, if restart is required we will need to ask supervisor to restart. Will return to this later (maybe create some issue for this - how do you think?)

Reply:
> Will return to this later (maybe create some issue for this - how do you think?)

I think we can return to it later - this is something that can be overridden by users via Streams attributes and we should probably provide a default of some kind. Let's add a GH issue for it and get back to it.

Reply:
Hi, did you do anything in this area? We're experiencing the issue that the stream is in a zombie state because all brokers are not available yet. Any ideas?
@IgorFedchenko 
@Aaronontheweb

Reply:
Can you open an issue? @realvictorpm